crates/eip7702/src/auth_list.rs

```diff
@@ -103,19 +108,26 @@ impl Authorization {
 pub struct SignedAuthorization {
     #[cfg_attr(feature = "serde", serde(flatten))]
     inner: Authorization,
-    #[cfg_attr(feature = "serde", serde(flatten))]
-    signature: Signature,
+    #[cfg_attr(feature = "serde", serde(rename = "yParity"))]
+    y_parity: U8,
+    r: U256,
+    s: U256,
```

Comment:
I order to write them to disk on reth we also need getters for all the fields

we need ensure that we can do:

https://github.com/paradigmxyz/reth/blob/cab76f2083be680afa4ddcff44420334dd0e71a5/crates/storage/codecs/src/alloy/authorization_list.rs#L42-L69